packages/fast-components-react-base/src/tabs/tab-item.tsx

```diff
@@ -0,0 +1,8 @@
+import * as React from "react";
+import Foundation, { HandledProps } from "../foundation";
+import { ITabItemHandledProps, ITabItemUnhandledProps } from "./tab-item.props";
+
+class TabItem extends Foundation<ITabItemHandledProps, ITabItemUnhandledProps, {}> {}
```

Comment:
This also isn't handling any of the handled props, or the unhandled props. Also, this has no render method, is that intentional?

Reply:
Yes, it's really just for semantic convenience, it does not even need to be a component it could be a `div` or any other element the user passes since we are only using it as a wrapper to find the contents and verify it contains both the `Tab` and `TabPanel`. I think it's more clear from an implementation perspective that this component exists however because I think it's how the user expects to compose it.

Reply:
At some level, I agree but it feels like we're mixing metaphors here. We're exposing a purpose-specific component that, unless *configured correctly*, will not work as expected. I would expect, if these sub-components we're available, that given a nesting structure it would do the right thing.

With this implementation though, it won't. Slots are the only thing that *actually* matter.

Reply:
Gotcha, I agree, I've changed this up to not use slots but check the component type as we are doing in `fast-layouts-react` for the `Grid` component.